tlf/id.go

```diff
@@ -257,3 +257,45 @@ func MakeRandomID(t Type) (ID, error) {
 	}
 	return id, nil
 }
+
+// MakeIDFromTeam makes a deterministic TLF ID from a team ID and an epoch
+// representing how many times a new TLF has been needed for this
+// team.  Returns NullID on failure.
+func MakeIDFromTeam(t Type, tid keybase1.TeamID, epoch byte) (ID, error) {
```

Comment:
I have a DeJa vu of writing this in `keybase1` and was gonna suggest removing it from `keybase1`, but I can't find it there. So maybe I didn't ...